tic-tac-toe/src/app/screens/Login/layout.js

```diff
@@ -1,10 +1,10 @@
 import React from 'react';
 import { Field, reduxForm } from 'redux-form';
+import PropTypes from 'prop-types';
 
 import style from './styles.scss';
 
 function LoginForm(props) {
```

Comment:
okey, we usually do it like this: `function LoginForm({ handleSubmit }) {`
So, we keep the props here.. it's the same you are doing, but we take a step. 
For classes.. you do have to use `const { myProp } = this.props;` because you don't have the option to define the props differently